README.md

```diff
@@ -79,50 +94,54 @@ Response:
 
 ### Composer
 
-```
+```bash
 composer require getkirby/kql
 ```
 
 ## Documentation
 
 ### API Endpoint
 
-KQL adds a new `query` API endpoint to your Kirby API. (i.e. https://yoursite.com/api/query) The endpoint requires authentication: https://getkirby.com/docs/guide/api/authentication
+KQL adds a new `query` API endpoint to your Kirby API (i.e. `yoursite.com/api/query`). This endpoint [requires authentication](https://getkirby.com/docs/guide/api/authentication).
 
 You can switch off authentication in your config at your own risk:
```

Comment:
This is a new feature, but it sounds more like an anti-feature. I think here we should tell the user what the risk actually is.

(@johannschopplich I know you didn't make any changes here, just leaving it as remark)